runbotics/runbotics-orchestrator-ui/src/src-landing/components/BlogLayout/BlogLayout.module.scss

```diff
@@ -8,12 +10,13 @@ $PADDING-BOTTOM: 30px;
     grid-template-rows: 1fr auto;
     grid-gap: $x-content-padding;
     max-width: $MAX_SECTION_CONTENT_WIDTH;
-
     margin: 0 auto;
-    padding: $HEADER_HEIGHT $x-content-padding $PADDING-BOTTOM
+    padding: 0 $x-content-padding $PADDING-BOTTOM
```

Comment:
<img width="334" alt="image" src="https://github.com/runbotics/runbotics/assets/69820069/3e3559d2-2322-4ccb-a262-9d9d86752e1b">

Reply:
There are 4 values:
![image](https://github.com/runbotics/runbotics/assets/44839807/307eb076-cb54-44ee-b745-266f9d78d4d4)
but last value was wrapped

Reply:
i've been meaning that the last one is redundant

padding: 0 $x-content-padding $PADDING-BOTTOM; 

gives the same result